embedded-minio/src/test/java/com/playtika/test/minio/EmbeddedMinioBootstrapConfigurationTest.java

```diff
@@ -52,23 +54,27 @@ public void shouldSuccessfullyWriteAndReadData() throws Exception {
         assertThat(content).isEqualTo("Hello Minio!");
     }
 
+    @Disabled("Current image doesn't support installing tc")
```

Comment:
can you please specify why tc package can not be installed there?

Reply:
The upstream project (MinIO) changed the docker images between the previous very old version and the current release version, it's Alpine based now, there is no tc available there.

Reply:
I'm not sure what do you mean by `there is no tc available there`. If the package manager was changed between versions, you can use the one that is available for the new distribution: https://github.com/Playtika/testcontainers-spring-boot/tree/develop/testcontainers-common/src/main/java/com/playtika/test/common/utils yum, apt-get and apk are already implemented